worksheet_C/README.md

```diff
@@ -0,0 +1,17 @@
+(a) Checks if any items on the list are repeated
+
+(b) It's taking a value from the list O(n) and checking it against every other value of the list O(n^2) 
+
+(c) It still checks every item on the list against every other item
+
+(d) It’s not going over results that it has already checked
+
+(e) Yes it’s still quadratic ((n^2)-n)/2
```

Comment:
✅ but explain why this is quadratic, i.e. n^2 dominates n, ignore the factor of 1/2